### ISSUE TITLE
Event notifications do not appear on Ubuntu 10.04.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [CarlitoBriganteNT/android-notifier#323]
```
Using Samsung Captivate on the phone. Test messages are displayed properly by 
the software. Battery status messages are also displayed. However, no other 
messages (SMS, Ring, etc) are displayed. I have confirmed that all events are 
checked to notify in the preferences. Below is log:

2010-12-17 05:52:05,652 INFO [ApplicationImpl] - Starting SWT
2010-12-17 05:52:07,098 INFO [ApplicationImpl] - Showing tray icon
2010-12-17 05:52:07,334 DEBUG [ServiceServerImpl] - Starting service server on 
port [10700]
2010-12-17 05:52:07,341 DEBUG [InetAddresses] - Looking for local address
2010-12-17 05:52:07,345 INFO [ApplicationImpl] - Starting [System default] 
broadcaster
2010-12-17 05:52:07,347 INFO [ApplicationImpl] - Starting [TCP] receiver
2010-12-17 05:52:07,978 DEBUG [InetAddresses] - Local address found
2010-12-17 05:52:11,428 INFO [ApplicationImpl] - Starting [UDP] receiver
2010-12-17 05:52:12,072 INFO [ApplicationImpl] - Adding to startup
2010-12-17 08:50:50,205 INFO [ApplicationImpl] - Starting SWT
2010-12-17 08:50:51,720 INFO [ApplicationImpl] - Showing tray icon
2010-12-17 08:50:51,860 DEBUG [ServiceServerImpl] - Starting service server on 
port [10700]
2010-12-17 08:50:51,871 DEBUG [InetAddresses] - Looking for local address
2010-12-17 08:50:51,873 INFO [ApplicationImpl] - Starting [System default] 
broadcaster
2010-12-17 08:50:51,874 INFO [ApplicationImpl] - Starting [TCP] receiver
2010-12-17 08:50:52,830 DEBUG [InetAddresses] - Local address found
2010-12-17 08:50:55,131 INFO [ApplicationImpl] - Starting [UDP] receiver
2010-12-17 08:50:55,291 INFO [ApplicationImpl] - Adding to startup
2010-12-17 09:01:03,665 INFO [NotificationManagerImpl] - Notification received: 
notificationId=39984721397593664, type=Battery
2010-12-17 09:01:14,307 INFO [NotificationManagerImpl] - Notification received: 
notificationId=1289818297419649, type=Ping
2010-12-17 09:01:49,123 INFO [NotificationManagerImpl] - Notification received: 
notificationId=1289818330883591, type=Ping
2010-12-17 09:16:41,025 INFO [ApplicationImpl] - Starting SWT
2010-12-17 09:16:41,699 INFO [ApplicationImpl] - Showing tray icon
2010-12-17 09:16:41,816 DEBUG [ServiceServerImpl] - Starting service server on 
port [10700]
2010-12-17 09:16:41,819 DEBUG [InetAddresses] - Looking for local address
2010-12-17 09:16:41,828 INFO [ApplicationImpl] - Starting [System default] 
broadcaster
2010-12-17 09:16:41,829 INFO [ApplicationImpl] - Starting [TCP] receiver
2010-12-17 09:16:42,030 DEBUG [InetAddresses] - Local address found
2010-12-17 09:16:42,503 INFO [ApplicationImpl] - Starting [UDP] receiver
2010-12-17 09:16:42,530 INFO [ApplicationImpl] - Adding to startup
2010-12-17 09:34:48,711 INFO [NotificationManagerImpl] - Notification received: 
notificationId=39984781678847706, type=Battery
2010-12-17 09:57:59,866 INFO [ApplicationImpl] - Starting SWT
2010-12-17 09:58:00,214 INFO [ApplicationImpl] - Showing tray icon
2010-12-17 09:58:00,315 DEBUG [ServiceServerImpl] - Starting service server on 
port [10700]
2010-12-17 09:58:00,328 DEBUG [InetAddresses] - Looking for local address
2010-12-17 09:58:00,334 INFO [ApplicationImpl] - Starting [System default] 
broadcaster
2010-12-17 09:58:00,334 INFO [ApplicationImpl] - Starting [TCP] receiver
2010-12-17 09:58:00,425 DEBUG [InetAddresses] - Local address found
2010-12-17 09:58:02,021 INFO [ApplicationImpl] - Starting [UDP] receiver
2010-12-17 09:58:02,122 INFO [ApplicationImpl] - Adding to startup
2010-12-17 10:48:44,889 INFO [NotificationManagerImpl] - Notification received: 
notificationId=39984913830738991, type=Battery
2010-12-17 10:53:53,925 INFO [ApplicationImpl] - Starting SWT
2010-12-17 10:53:54,215 INFO [ApplicationImpl] - Showing tray icon
2010-12-17 10:53:54,336 DEBUG [ServiceServerImpl] - Starting service server on 
port [10700]
2010-12-17 10:53:54,339 DEBUG [InetAddresses] - Looking for local address
2010-12-17 10:53:54,351 INFO [ApplicationImpl] - Starting [System default] 
broadcaster
2010-12-17 10:53:54,351 INFO [ApplicationImpl] - Starting [TCP] receiver
2010-12-17 10:53:54,753 DEBUG [InetAddresses] - Local address found
2010-12-17 10:53:54,940 INFO [ApplicationImpl] - Starting [UDP] receiver
2010-12-17 10:53:54,984 INFO [ApplicationImpl] - Adding to startup
2010-12-17 11:37:50,001 DEBUG [TextNotificationParser] - Got notification but 
it has less fields than expected, maybe it's encrypted, ignoring
2010-12-17 11:37:50,002 DEBUG [TextNotificationParser] - Got notification but 
it has less fields than expected, maybe it's encrypted, ignoring
2010-12-17 11:37:50,020 DEBUG [TextNotificationParser] - Got notification but 
it has less fields than expected, maybe it's encrypted, ignoring
2010-12-17 11:37:50,038 DEBUG [TextNotificationParser] - Got notification but 
it has less fields than expected, maybe it's encrypted, ignoring
2010-12-17 11:37:50,067 DEBUG [TextNotificationParser] - Got notification but 
it has less fields than expected, maybe it's encrypted, ignoring
2010-12-17 11:37:50,079 DEBUG [TextNotificationParser] - Got notification but 
it has less fields than expected, maybe it's encrypted, ignoring
2010-12-17 11:41:42,666 DEBUG [TextNotificationParser] - Got notification but 
it has less fields than expected, maybe it's encrypted, ignoring
2010-12-17 11:41:42,686 DEBUG [TextNotificationParser] - Got notification but 
it has less fields than expected, maybe it's encrypted, ignoring
2010-12-17 11:41:42,706 DEBUG [TextNotificationParser] - Got notification but 
it has less fields than expected, maybe it's encrypted, ignoring
2010-12-17 11:41:42,726 DEBUG [TextNotificationParser] - Got notification but 
it has less fields than expected, maybe it's encrypted, ignoring
2010-12-17 11:41:42,746 DEBUG [TextNotificationParser] - Got notification but 
it has less fields than expected, maybe it's encrypted, ignoring
2010-12-17 13:12:51,891 INFO [NotificationManagerImpl] - Notification received: 
notificationId=39985171370716253, type=Battery
2010-12-17 13:14:27,408 DEBUG [TextNotificationParser] - Got notification but 
it has less fields than expected, maybe it's encrypted, ignoring
2010-12-17 13:14:27,409 DEBUG [TextNotificationParser] - Got notification but 
it has less fields than expected, maybe it's encrypted, ignoring
2010-12-17 13:14:27,410 DEBUG [TextNotificationParser] - Got notification but 
it has less fields than expected, maybe it's encrypted, ignoring
2010-12-17 13:14:27,411 DEBUG [TextNotificationParser] - Got notification but 
it has less fields than expected, maybe it's encrypted, ignoring
2010-12-17 13:14:27,412 DEBUG [TextNotificationParser] - Got notification but 
it has less fields than expected, maybe it's encrypted, ignoring
2010-12-17 13:14:27,413 DEBUG [TextNotificationParser] - Got notification but 
it has less fields than expected, maybe it's encrypted, ignoring
2010-12-17 13:14:27,413 DEBUG [TextNotificationParser] - Got notification but 
it has less fields than expected, maybe it's encrypted, ignoring
2010-12-17 13:14:27,414 DEBUG [TextNotificationParser] - Got notification but 
it has less fields than expected, maybe it's encrypted, ignoring
2010-12-17 13:14:27,427 DEBUG [TextNotificationParser] - Got notification but 
it has less fields than expected, maybe it's encrypted, ignoring
2010-12-17 13:14:27,446 DEBUG [TextNotificationParser] - Got notification but 
it has less fields than expected, maybe it's encrypted, ignoring
2010-12-17 13:14:27,466 DEBUG [TextNotificationParser] - Got notification but 
it has less fields than expected, maybe it's encrypted, ignoring
2010-12-17 13:14:27,487 DEBUG [TextNotificationParser] - Got notification but 
it has less fields than expected, maybe it's encrypted, ignoring
2010-12-17 13:14:27,505 DEBUG [TextNotificationParser] - Got notification but 
it has less fields than expected, maybe it's encrypted, ignoring
2010-12-17 13:14:27,525 DEBUG [TextNotificationParser] - Got notification but 
it has less fields than expected, maybe it's encrypted, ignoring
2010-12-17 13:14:27,546 DEBUG [TextNotificationParser] - Got notification but 
it has less fields than expected, maybe it's encrypted, ignoring
2010-12-17 13:14:27,564 DEBUG [TextNotificationParser] - Got notification but 
it has less fields than expected, maybe it's encrypted, ignoring
2010-12-17 13:14:27,586 DEBUG [TextNotificationParser] - Got notification but 
it has less fields than expected, maybe it's encrypted, ignoring
2010-12-17 13:40:56,222 INFO [NotificationManagerImpl] - Notification received: 
notificationId=39985221567506956, type=Battery
2010-12-17 13:50:02,526 INFO [NotificationManagerImpl] - Notification received: 
notificationId=39984826122210757, type=SMS
2010-12-17 14:25:57,426 INFO [NotificationManagerImpl] - Notification received: 
notificationId=39985302032520990, type=Battery
2010-12-17 14:41:10,193 INFO [NotificationManagerImpl] - Notification received: 
notificationId=39984947599647548, type=Ring
2010-12-17 14:41:49,214 INFO [NotificationManagerImpl] - Notification received: 
notificationId=39985017347306055, type=Ring
2010-12-17 14:49:54,988 INFO [NotificationManagerImpl] - Notification received: 
notificationId=39985344905104970, type=Battery
2010-12-17 16:50:05,501 INFO [NotificationManagerImpl] - Notification received: 
notificationId=39985559642414505, type=Battery
2010-12-17 18:38:44,070 INFO [NotificationManagerImpl] - Notification received: 
notificationId=39985370696938500, type=Ring
2010-12-17 18:39:43,986 INFO [NotificationManagerImpl] - Notification received: 
notificationId=39985755634662423, type=Battery
```

Original issue reported on code.google.com by `docalt...@gmail.com` on 20 Dec 2010 at 4:53

**Comment by GoogleCodeExporter** [comment]
```
Maybe Windows is just not showing popup ballons sometimes, take a look at my 
comment in issue 114: 
http://code.google.com/p/android-notifier/issues/detail?id=114#c8

Custom themes may cause this too.
```

Original comment by `lehph...@gmail.com` on 20 Dec 2010 at 6:44
- Added labels: **OpSys-Windows**

**Comment by GoogleCodeExporter** [comment]
```
I'm using Ubuntu 10.04, not Windows. All other notifications show up correctly, 
and the test ping notification appears correctly every time, so I don't think 
it's a problem with the notification system.
```

Original comment by `docalt...@gmail.com` on 20 Dec 2010 at 8:10

**Comment by GoogleCodeExporter** [comment]
```
Sorry, next time please make sure you follow the template for new issues.

Does it always happen? Does it happen only after you get some other 
notification? What Gnome theme are you using, what is the output of "java 
-version" in terminal and what are the steps to make this happen?
```

Original comment by `lehph...@gmail.com` on 20 Dec 2010 at 8:28
- Added labels: **OpSys-Linux**
- Removed labels: **OpSys-Windows**

**Comment by GoogleCodeExporter** [comment]
```
Oh my. It all of a sudden started working.

Well, thank you for your help, as well as the magic fixit.

```

Original comment by `docalt...@gmail.com` on 20 Dec 2010 at 11:19

**Comment by GoogleCodeExporter** [comment]
```
Cool :) if it happens again, create a new issue with answers for those 
questions.
```

Original comment by `lehph...@gmail.com` on 21 Dec 2010 at 2:28
- Changed state: **Invalid**
